python/dgl/transform.py

```diff
@@ -581,6 +581,28 @@ def partition_graph_with_halo(g, node_part, num_hops):
         subg_dict[i] = subg
     return subg_dict
 
+def metis_partition(g, k):
```

Comment:
Is it better to set the partition id to node frame, and use another function to split into dict? Since I would assume the partition result could be used somewhere else.

Reply:
the partition result is still preserved.